CHANGELOG.md

```diff
@@ -4,6 +4,18 @@
 
 Version <dev> introduces the option to store tracer state on the thread-level and hardens the browser agent insertion logic to better proactively anticipate errors.
 
+- **Feature: Add instrumentation for ruby-openai**
+
+  Instrumentation has been added for the [ruby-openai](https://github.com/alexrudall/ruby-openai) gem, supporting versions 3.4.0 and higher (PR#2442)[https://github.com/newrelic/newrelic-ruby-agent/pull/2442]. While ruby-openai instrumentation is enabled by default, the configuration option `ai_monitoring.enabled` is disabled by default and controls all AI monitoring. `ai_monitoring.enabled` must be set to `true` in order to receive ruby-openai instrumentation.
+
+  Calls to embedding and chat completion endpoints are automatically traced. Two new APIs allow users to record additional information on events:
```

Comment:
Maybe this part should start a new section, like `Add LLM API methods` or something? since it'll be able to be used for bedrock too once we release the bedrock stuff

Reply:
I like that @tannalynn! Its been updated